generators/app/templates/stories/index.jsx

```diff
@@ -3,6 +3,21 @@ import React from 'react';
 import { storiesOf, action, linkTo } from '@kadira/storybook';
 import <%= jsxFileName %> from '../src/<%= jsxFileName %>';
 
+import { checkA11y } from 'storybook-addon-a11y';
+import 'storybook-addon-i18n-tools';
```

Comment:
If we're using the i18n tools (which I think we should), we need to make sure to remove the 'dir' attribute we're setting here: https://github.com/cerner/generator-terra-module/blob/master/generators/app/templates/.storybook/head.html#L5

The addon puts the 'dir' tag to the body element.  If we leave the one at root, very, very weird behavior will result with both 'ltr' and 'rtl' styles are active at the same.